### DISCUSSION TITLE
Image fetching issue  <title>

### DISCUSSION THREAD

**Discussion by Axhutoxh** [community/community#41580] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Not able to fetch image from git hub repo

**Comment by radar-blip** [comment, marked as answer]
Need more info. Which repo would that be?

**Reply by radar-blip** [reply]
Images are showing fine.

![imagess](https://user-images.githubusercontent.com/39196767/207242646-77579928-dc7c-4d3a-b3ea-087dec5e163b.png)

**Comment by Axhutoxh** [comment]
in my case currently repo is (https://github.com/Axhutoxh/javascript/blob/main/study/chapter1/README.md) in readmeFile . i have used gif,png.
it was working fine till previous month. but from current month readme file not able to fetch img and gif

**Reply by DSDmark** [reply]
Hi @Axhutoxh,

can you tall us which GIF file you want and add link of your GIF file?

**Reply by DSDmark** [reply]
On my side, everything's is fine. I can see every image on that README

**Reply by radar-blip** [reply]
I can see everything fine also. All jpg and png images are visible

**Comment by Axhutoxh** [comment]
![image](https://user-images.githubusercontent.com/40849321/207248202-7f1f3109-8de2-445c-8166-488feac1f01d.png)
@linuxlawson @DSDmark  in my case i get

**Reply by DSDmark** [reply]
@Axhutoxh, try to clear your browsers history and cookies. And also if you use some extensions in browser, then disable it.

**Reply by DSDmark** [reply]
And also check your browser updates. If it not works then try to with reinstall it.

**Reply by Axhutoxh** [reply]
@DSDmark but i am not able to get image in tablet and mobile also.

**Reply by DSDmark** [reply]
@Axhutoxh, then, Check your internet connection. If you are experiencing a slow or unstable internet connection, it could affect your ability to view images on GitHub.

**Reply by Axhutoxh** [reply]
@DSDmark 
![image](https://user-images.githubusercontent.com/40849321/207251202-1cd1bff4-4bd1-4a7f-9f54-574bbe744b81.png)

**Reply by DSDmark** [reply]
Dude... I have nothing to tell you. The last things you can do to connect with GitHub support:- https://support.github.com/

**Reply by DSDmark** [reply]
@Axhutoxh And also you can take advice to Mr.@linuxlawson. He may know something's about that.

**Reply by radar-blip** [reply]
I got nothing. I checked code and cant find anything wrong. Images are still visible to me. 😕

**Reply by Axhutoxh** [reply]
@DSDmark @linuxlawson thanks for reach out  , got the issue , it is due to my server  in my case if i use jio network service i didn't get the image . if i switch to other network then i am able to get images as expected.

**Reply by Axhutoxh** [reply]
@linuxlawson @DSDmark but still i have a confusion i considered above problem as an issue or not 😂😂

**Reply by radar-blip** [reply]
Not a problem. Glad you got it worked out. 👍🏻

**Reply by DSDmark** [reply]
by the way, I'm facing same network problem with Jio while ago. I don't Jio why doing this with us. :smiling_face_with_tear:

**Reply by Prakash4844** [reply]
Changing DNS fixes it.

**Comment by Prakash4844** [comment]
I am also getting same issue on Jio Network, and I am linking screenshot. [On Local](https://i.imgur.com/0SM6Un3.png) and on [GitHub with Jio network ](https://i.imgur.com/EXvWnn2.png) and here's a link to [repo](https://github.com/Prakash4844/Dressingnity-Ecommerce-Website) see if you can see the images.
